### PR TITLE
Use displaynames in pop sens plot

### LINKED ISSUE #214: Crash in Individual Sensitivity Analysis
Execute `Workflow.R` from the example attached [bug01.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/4740657/bug01.zip)
```
Error in grid.Call.graphics(C_setviewport, vp, TRUE) : 
  non-finite location and/or size for viewport
```

Note: sensitivity values do not correspond to the pkml model attached, because I had to replace the original model file with a dummy due to privacy reasons. Workflow.R just uses precalculated sensitivity values for plotting

### LINKED ISSUE #218: MeanModel Sensitivity plot crashes for some PK-Parameters.
Sensitivity plot crashes for some PK-Parameters.
E.g. for [03.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/4755430/03.zip):
`Values 'AUC_inf_tD1_norm' are not included in parent values: 'C_max', 'C_max_tD1_tD2', 'C_max_tDLast_tEnd', 't_max', 't_max_tD1_tD2', 't_max_tDLast_tEnd', 'C_trough_tD2', 'C_trough_tDLast', 'AUC_tD1_tD2', 'AUC_tDLast_minus_1_tDLast', 'AUC_inf_tD1', 'AUC_inf_tDLast', 'MRT', 'Thalf', 'Thalf_tDLast_tEnd'."`

Where does this _parent values_ list come from? It containes only 15 out of 22 calculated PK-Parameters (s. **Report\PKAnalysisResults\SimSet-PKAnalysisResults.csv**)

### LINKED ISSUE #257: Remove tracelib debug output
grafik

### PR DESCRIPTION
**Pull request by abdullahhamadeh**
Fixes #214 
Fixes #218 
Fixes #257 
Use displayName for output and pkparameter in pop sensitivity plot output captions

### CODE REVIEW

File: R/utilities-sensitivity-analysis.R [hunk 1]
```diff
@@ -623,6 +635,29 @@ plotTornado <- function(data,
...
+lookupPKParameterDisplayName <- function(output,pkParameter){
+  pkParameterDisplayName <- NULL
+
+  for (pk in output$pkParameters){
+    # print("-----")
+    # print(paste("pkParameter is",pkParameter))
+    # print(paste("pk object is"))
+    # print(pk)
+    # print("^^^^^")
+    if ( pkParameter %in% pk$pkParameter ) {
```
Comment (msevestre) [inline R/utilities-sensitivity-analysis.R:652]: Should be == no?

File: R/utilities-sensitivity-analysis.R [hunk 2]
```diff
@@ -623,6 +635,29 @@ plotTornado <- function(data,
...
+      pkParameterDisplayName <- pk$displayName
```
Comment (msevestre) [inline R/utilities-sensitivity-analysis.R:653]: If found, return directly. No need to break

File: R/utilities-sensitivity-analysis.R [hunk 3]
```diff
@@ -623,6 +635,29 @@ plotTornado <- function(data,
...
+#' @title lookupPKParameterDisplayName
+#' @param output an Output object
+#' @param pkParameter a string with the name of the pkParameter from the population sensitivity results index file
+#' @return the display name of the input pk parameter or the string pkParameter itself if no display name found
+lookupPKParameterDisplayName <- function(output,pkParameter){
+  pkParameterDisplayName <- NULL
+
+  for (pk in output$pkParameters){
+    # print("-----")
+    # print(paste("pkParameter is",pkParameter))
```
Comment (msevestre) [inline R/utilities-sensitivity-analysis.R:648]: Remove comments

File: R/utilities-sensitivity-analysis.R [hunk 4]
```diff
@@ -662,7 +697,10 @@ plotPopulationSensitivity <- function(structureSets,
     saResultIndexFiles[[populationName]] <- structureSet$popSensitivityAnalysisResultsIndexFile
     simulationList[[populationName]] <- simulation
 
-    for (op in outputPaths) {
+    for (output in structureSet$simulationSet$outputs) {
+      op <- output$path
```
Comment (msevestre) [inline R/utilities-sensitivity-analysis.R:701]: Rename to outputPath if it's needed at all

File: R/utilities-tracelib.R [hunk 5]
```diff
@@ -3,35 +3,35 @@ tracelibCheck <- function() {
...
-  print("tStoreFileMetadata")
+  #print("tStoreFileMetadata")
   if (tracelibCheck()) {
     tracelib::tStoreFileMetadata(...)
   }
 }
 
 re.tStartAction <- function(...) {
-  print("tStartAction")
+  #print("tStartAction")
```
Comment (msevestre) [inline R/utilities-tracelib.R:27]: Do we need those print? If not, remove

File: R/utilities-sensitivity-analysis.R [hunk 6]
```diff
@@ -749,7 +809,9 @@ plotPopulationSensitivity <- function(structureSets,
             "Value" = missingSensivitity,
             "Quantile" = qu,
             "individualId" = id,
```
Comment (msevestre) [inline R/utilities-sensitivity-analysis.R:811]: Shouldn't that be IndividualId (capital I)

Comment (abdullahhamadeh) [reply R/utilities-sensitivity-analysis.R]: Fixed

File: R/utilities-sensitivity-analysis.R (hunk 4 above)
Comment (abdullahhamadeh) [reply R/utilities-sensitivity-analysis.R]: Fixed

File: R/utilities-sensitivity-analysis.R (hunk 3 above)
Comment (abdullahhamadeh) [reply R/utilities-sensitivity-analysis.R]: Fixed

File: R/utilities-sensitivity-analysis.R (hunk 2 above)
Comment (abdullahhamadeh) [reply R/utilities-sensitivity-analysis.R]: Fixed

File: R/utilities-sensitivity-analysis.R (hunk 1 above)
Comment (abdullahhamadeh) [reply R/utilities-sensitivity-analysis.R]: Fixed

File: R/utilities-tracelib.R (hunk 5 above)
Comment (abdullahhamadeh) [reply R/utilities-tracelib.R]: Fixed

**Pull request merged**